crates/turbopack-ecmascript/src/analyzer/builtin.rs

```diff
@@ -465,6 +465,17 @@ pub fn replace_builtin(value: &mut JsValue) -> bool {
                 true
             }
         }
+        JsValue::Tenary(_, test, cons, alt) => {
+            if test.is_truthy() == Some(true) {
+                *value = take(cons);
+                true
+            } else if test.is_falsy() == Some(true) {
```

Comment:
This is just `test.is_truty() == Some(false)`, right? We can this into a branch above and save some cycles.

Reply:
I think so, this is borrowing existing references from elsewhere (I recall it's binary operation or something)

Reply:
Yeah this can be a match on is_truty